tests/installation/first_boot.pm

```diff
@@ -48,7 +48,8 @@ sub run {
         return;
     }
     elsif ((get_var('DESKTOP', '') =~ /textmode|serverro/) || get_var('BOOT_TO_SNAPSHOT')) {
-        assert_screen('linux-login', $boot_timeout) unless check_var('ARCH', 's390x');
+        my $tty = get_root_console_tty;
+        assert_screen("tty$tty-selected", $boot_timeout) unless check_var('ARCH', 's390x');
```

Comment:
This is firstboot. After the first boot in textmode, tty1 is shown. And tty1 is not the root-console in openQA.

```perl
sub get_root_console_tty {                                                                                                                                                                                                                    
    return (!is_sle('<15') && !is_caasp && !check_var('VIRSH_VMM_TYPE', 'linux')) ? 6 : 2;                                                                                                                                                    
}
```

This has to be

```perl
assert_screen('tty1-selected', $boot_timeout) unless check_var('ARCH', 's390x');
```

Reply:
since first_boot.pm got changed (re-factory), I checked the function which uses linux-login and provide new test runs, please check and leave a comment, thanks.